### ISSUE TITLE
Cannot read property getNodeFromOffset of null

### ISSUE THREAD

**Issue by felixfbecker** [redhat-developer/yaml-language-server#18]
I am getting this error in the console a lot:
```
[Error - 11:20:19] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'getNodeFromOffset' of null
  Code: -32603 
```

**Comment by JPinkney** [comment]
Is there any specific cases that it is happening in?

**Comment by JPinkney** [comment]
This should be fixed as of 0.0.7
